kie-api/src/main/java/org/kie/api/runtime/KieSession.java

```diff
@@ -129,9 +129,19 @@
     void submit(AtomicAction action);
 
     /**
+     * Returns a runtime for the given interface
+     *
+     * @param cls
+     * @param <T>
+     * @return
+     */
+    <T> T getKieRuntime(Class<T> cls);
```

Comment:
Could you please provide more info about what the method returns? From the currently description I don't really understand what should I pass into the method.

Also, it would be great to either add some description for the params, or remove them from the javadoc.